src/logging.cpp

```diff
@@ -89,17 +92,27 @@ static void register_loggers(const std::string& log_level, std::vector<spdlog::s
     spdlog::set_default_logger(serving_logger);
 }
 
-void configure_logger(const std::string log_level, const std::string log_path) {
+void configure_logger(const std::string& log_level, const std::string& log_path) {
     static bool wasRun = false;
-    if (wasRun)
+    if (wasRun) {
+        SPDLOG_WARN("Tried to configure loggers twice. Keeping previous settings.");
         return;
+    }
     wasRun = true;
     std::vector<spdlog::sink_ptr> sinks;
     sinks.push_back(std::make_shared<spdlog::sinks::stdout_sink_st>());
     if (!log_path.empty()) {
         sinks.push_back(std::make_shared<spdlog::sinks::basic_file_sink_mt>(log_path));
     }
     register_loggers(log_level, sinks);
+#if (MEDIAPIPE_DISABLE == 0)
+    if (log_level == "DEBUG" || log_level == "TRACE")
+        FLAGS_minloglevel = google::INFO;
+    else if (log_level == "WARNING")
+        FLAGS_minloglevel = google::INFO;
+    else  // ERROR, WARNING, FATAL
```

Comment:
We have WARNING in a condition above - having it among the ERROR and FATAL in this comment is confusing.